python/ray/tune/suggest/suggestion.py

```diff
@@ -382,7 +406,25 @@ def __init__(self,
         super(ConcurrencyLimiter, self).__init__(
             metric=self.searcher.metric, mode=self.searcher.mode)
 
+    def set_max_concurrency(self, max_concurrent: int) -> bool:
+        # Determine if this behavior is acceptable, or if it should
+        # raise an exception.
+        self.max_concurrent = max_concurrent
+        return True
```

Comment:
Feedback here would be appreciated @krfricke @xwjiang2010

Reply:
I like it!